rest_framework/schemas/openapi.py

```diff
@@ -500,12 +500,17 @@ def map_field(self, field):
                 'format': 'binary'
             }
 
+        # Also handles serializers.HStoreField,
+        if isinstance(field, serializers.DictField):
+            return {
+                "type": "object",
+                "additionalProperties": self.map_field(field.child),
+            }
```

Comment:
Support `help_text`:

```suggestion
        if isinstance(field, DictField):
            schema = {
                "type": "object",
                "additionalProperties": self.map_field(field.child),
            }
            if field.help_text:
                schema["description"] = field.help_text
            return schema
```